CHANGES

```diff
@@ -220,6 +220,13 @@ regressions due to the Sphinx migration), and user message improvements
 (for details, see the 8.8.1 milestone at
 https://github.com/coq/coq/milestone/13?closed=1).
 
+Notations
+
+- New command "Declare Scope" to explicitly declare a scope name
+  before any use of it. Implicit declaration of a scope at the time of
+  "Bind Scope", "Delimit Scope", "Undelimit Scope", or "Notation" is
+  deprecated.
+
```

Comment:
I am much afraid that this entry is placed in the wrong place, please fix @herbelin

Reply:
Right! Moved to post-8.8.2 (BTW, who is supposed to eventually restructurate the `CHANGES`, e.g. to glue the two post-8.8.2 sections `Notations`, etc., shall it be @silene?)